lib/SILOptimizer/Mandatory/TFDeabstraction.cpp

```diff
@@ -1693,8 +1693,16 @@ void TFDeabstraction::checkAttributesAndFormGraphOps() {
     // If this is a normal tensor operation, validate it and transform it into a
     // graphOp instruction.
     if (TFStrictDeabstraction) {
-      if (auto opInfo = SILTensorOpInfo::decode(inst))
+      if (auto opInfo = SILTensorOpInfo::decode(inst)) {
+        // Do not translate this special inst into a graph op, since otherwise
+        // it gets removed by the performance pipeline.
```

Comment:
The performance optimizer is deleting our graph_op instructions?  If so, that's a serious problem.  It looks like the fix for this is to change SILNodes.def from:

MULTIPLE_VALUE_INST(GraphOperationInst, graph_op,
                    SILInstruction, None, DoesNotRelease)

to:

MULTIPLE_VALUE_INST(GraphOperationInst, graph_op,
                    SILInstruction, MayHaveSideEffects, DoesNotRelease)

Reply:
Done. I'm still not transforming the tfop for "ConfigureTPU/ConfigureGPU" to a graph_op, because we expect the deabstraction pass to consume/delete that inst, once we merge partition pass into deabstraction.

Reply:
SG, please update the comment above.

Reply:
Done.